app/components/identity-steps/step-seven.hbs

```diff
@@ -1,36 +1,60 @@
 <div data-test-verification-page class='verification-page'>
-  {{#if (eq this.login.userData.profileStatus 'PENDING')}}
-    <div
-      data-test-verification-heading-icon
-      class='verification-page__heading-Icon'
-    >
-      <h3
-        data-test-verification-heading
-        class='verification-page__heading heading__h3'
+  {{#each this.currentProfileStatus as |currentStatus|}}
+    {{#if (eq this.login.userData.profileStatus currentStatus.status)}}
+      <div
+        data-test-verification-heading-icon
+        class='verification-page__heading-Icon'
       >
-        Pending
-      </h3>
-      <FaIcon @icon='hourglass-half' @size='lg' class='hourglass-icon' />
-    </div>
-    <div
-      data-test-verification-description
-      class='verification-page__description'
-    >
-      <p data-test-verification-description1>
-        Refresh to Check Verification Status
-      </p>
-      <p data-test-verification-description2>
-        Your Profile Service Linked with Real Dev Squad Service
-      </p>
-    </div>
-    <div class='verification-page__button'>
+        <h3
+          data-test-verification-heading
+          class='verification-page__heading heading__h3'
+        >
+          {{currentStatus.heading}}
+        </h3>
+        <FaIcon
+          @icon={{currentStatus.icon}}
+          @size='lg'
+          class='{{currentStatus.icon}}-icon'
+        />
+      </div>
+      <div
+        data-test-verification-description
+        class='verification-page__description'
+      >
+        {{#if (eq currentStatus.status 'PENDING')}}
+          <p data-test-verification-description1>
+            Refresh to Check Verification Status
+          </p>
+          <p data-test-verification-description2>
+            Your Profile Service Linked with Real Dev Squad Service
+          </p>
+        {{else if (eq currentStatus.status 'BLOCKED')}}
+          <p data-test-verification-description1>

```

Comment:
The data-test attributes can be improved. 
If using number suffix, `description-1` might be better for readability.
Also, the container div can be `description-container`

Reply:
Done with changes